### ISSUE TITLE
Optimize how Webmaker and teach.moz.org show up on mozilla.org/contribute

### ISSUE THREAD

**Issue by OpenMatt** [MozillaFoundation/plan#290]
https://www.mozilla.org/en-US/contribute/

@adamlofting would be good to know how it's performing, and metrics-based recommendations for improvements.

**Comment by adamlofting** [comment]
Noted :smile: 
I'll investigate who has data for this, and what it looks like.

**Comment by OpenMatt** [comment]
@adamlofting what do you think about this one? should we still do it? and if so, what's a timeframe that makes sense? I'm a bit worried that a major contributor pipeline may be falling through the cracks.

**Comment by adamlofting** [comment]
I'm waiting to hear back on numbers, but we should definitely keep an eye on this user journey. We'll likely need to update a bunch of the messaging and links given the move around in product and CTAs within webmaker.

I'll re-scope this ticket to cover 'investigating the numbers', then once we have data we can file tickets to make appropriate updates. 

This is probably a strong channel to kick off traffic to the Mentor Networks site.

**Comment by adamlofting** [comment]
@OpenMatt here are our numbers:

https://docs.google.com/a/mozillafoundation.org/spreadsheets/d/1d4qnRlqhIPM-rnk8oOQZTQETpyqfVFVHqCvo_tEE7IM/edit#gid=0

For the `Teaching` > `Teach the web (webmaker)` route, we see about 100 inquiries per month, but lowish click-through-rate in reply to the auto-responder emails.

It would be interesting to know if we follow up with these inquiries personally? At this volume it feels manageable as a route to on-boarding. Even if we just find out what these people expected when they volunteered to Teach the Web.

At a first glance the language and options to get from the /contribute page into a MoFo 'channel' feel a bit unclear for someone who doesn't know what we do.

Once we have a new Teach site as a destination for these people I think it's a channel worth 'optimizing' for.

**Comment by adamlofting** [comment]
Shall we close this 'get the data' ticket, and then meet to talk about / plan future iterations on this user journey?

**Comment by chrislarry33** [comment]
@adamlofting Yes let's do that. Matt is on vacation for another week. So we can pick up then.

**Comment by OpenMatt** [comment]
Going to move this forward to Mar 27 so we remember to follow up.

**Comment by OpenMatt** [comment]
@adamlofting @hannahkane: I'm going to move this to April 24 so we can take it up in relation to promotion of teach.mozilla.org

**Comment by OpenMatt** [comment]
re-titled this ticket -- blocked on teach.moz.org soft launch. once it's up, we should re-visit the issues we listed above.

**Comment by edrushka** [comment]
@HPaulJohnson @KevZawacki @edrushka + George Roter and Rebecca Davies - flagging all of us since we're working with MoCo on Moz.org refresh (which may include Contribute page)

and @valianttry who works closely with Tiles team

**Comment by OpenMatt** [comment]
@hannahkane let's fold into some of the experiments we want to run on teach.moz.org as per our convo in program manager meeting
